Automattic/wp-calypso
# Launchpad: hide "edit design" overlay button for all flows

Users tend to hover on the previewer to check their theme, but the Edit design button covers the whole area, which might seem to be blocked.


## Proposed Changes

- Remove "edit on hover" functionality on all onboarding flows, and from the component itself as it was used only for onboarding.

## Testing Instructions

1. Start any new launchpad enabled site, here are couple examples:
   - Newsletter http://calypso.localhost:3000/setup/newsletter _(flow didn't have the "edit" button before PR either)_
   - Link-in-Bio http://calypso.localhost:3000/setup/link-in-bio
   - VideoPress http://calypso.localhost:3000/setup/videopress
   - Free http://calypso.localhost:3000/setup/free
   - Build http://calypso.localhost:3000/start ( select `Other` as the site goal )
   - Write http://calypso.localhost:3000/start ( select `Write & Publish` as the site goal ) _(flow didn't have the "edit" button before PR either)_
1. Navigate to the Launchpad by completing onboarding
1. Hover the site preview; nothing should show up

**Before**

<img width="1446" alt="Screenshot 2023-05-11 at 14 37 02" src="https://github.com/Automattic/wp-calypso/assets/87168/7b8ed915-4cbf-4e75-8447-451c2846f3b0">


**After**

<img width="1447" alt="Screenshot 2023-05-11 at 14 37 29" src="https://github.com/Automattic/wp-calypso/assets/87168/c0df8909-5cf2-42f0-944e-5e320ba040ce">


## Pre-merge Checklist
